```
catch_backtrace()
```

Obtenez la trace de la pile de l'exception actuelle, Ã  utiliser dans les blocs `catch`.
